osu.Game/Screens/OnlinePlay/Multiplayer/Spectate/MultiSpectatorScreen.cs

```diff
@@ -108,6 +132,15 @@ private void load()
                     leaderboard.AddClock(instance.UserId, instance.GameplayClock);
 
                 leaderboardContainer.Add(leaderboard);
+
+                if (leaderboard.TeamScores.Count >= 2)
+                {
+                    LoadComponentAsync(new MatchScoreDisplay
+                    {
+                        Team1Score = { BindTarget = leaderboard.TeamScores.First().Value },
+                        Team2Score = { BindTarget = leaderboard.TeamScores.Last().Value },
```

Comment:
This is a little dicey (works for two and only exactly two teams), but I suppose it's good enough for now...

Would maybe change the conditional to only do this when there are exactly 2 teams.